aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -573,7 +573,7 @@ It's often convenient to use `<select>` with an `enum` property and generate the
 
 Sample:
 
-[!code-csharp[](working-with-forms/sample/final/ViewModels/CountryEnumViewModel.cs?range=3-7)]
+[!code-csharp[](working-with-forms/sample/final/ViewModels/CountryEnumViewModel.cs?range=3-6)]
```

Comment:
Fixes the range (remove starting bracket):

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/c290c73a-dced-46bc-82cb-26179b7daa8e)